udata/tests/api/test_datasets_api.py

```diff
@@ -1325,3 +1326,27 @@ def test_resource_types_list(self):
         response = self.get(url_for('api.resource_types'))
         self.assert200(response)
         self.assertEqual(len(response.json), len(RESOURCE_TYPES))
+
+
+@pytest.mark.usefixtures('clean_db')
+class DatasetSchemasAPITest:
```

Comment:
Wrote tests for my new endpoint using the `pytest` syntax. Other classes inherited from `APITestCase`, not compatible with `pytest`.